_summaries/wu2018group_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{wu2018}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # SShivang
+score: # 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Core Idea:
+
+The batch norm introduced an optimization in which the outputs of the layers were normalized for a particular batch (scaled and recentered). This lead to significant improvement in performance for a lot of deep learning tasks (one of the most important improvements). However with smaller batch sizes, this technique doesn't work as well. This may be the case if it is impossible to have large batch sizes (models such as video classification models) and you have to train on a smaller batch size. This paper introduces a group norm which works on smaller batches and has consistent improvements of performance across batch sizes. Group norm breaks down does normalization of the input layer by breaking channels into groups.
+
+Technical Realization:
+
+![Performance of Batch Norm vs Group Norm](wu2018group_2_1.png)
+
+Batch norms are highly dependent on batches. However, for tasks such as image segmentation, object detection, and video classification, models train on smaller batch sizes as their inputs are so large. In this case batch norm is less useful.
+
+![All normalization techniques](wu2018group_2_6.png)
+
+ This paper takes inspiration from techniques such as layer norms and instance norms, which are batch independent. However, Instance Norm goes to far as it doesn't leverage all the information across channels. Therefore the it loses too much statisitcal properties that other techniques employ. In relation to layer norm, group norm does better since the model has the flexibility of learning different distribution for each group and doesn't assume that the statisitcal distribution accross all channels is the same (in case of layer norm). This leads to improvements in representations. Group Normalization does better than all of the techniques in this respect as it is batch independent, and it has a good trade-off between how many channels it normalizes across (exploiting channel dependence while still not assuming all the channels have same distributional characterisitcs).
```

Comment:
I feel like you should go more into the technical realization here. An important point is that they do not apply GN directly on the image inputs, but in some cases, after the convolution process. They apply GN on the features generated after the convolution layer in several experiments in the paper. 
You could discuss why GN works better in the performance section

Reply:
Added that realization to this section.